inference-engine/tests/functional/plugin/myriad/common/myriad_common_test_utils.hpp

```diff
@@ -4,18 +4,10 @@
 
 #pragma once
 
-#include <cstdlib>
-#include <iostream>
-
 namespace CommonTestUtils {
 namespace vpu {
 
-bool CheckMyriad2() {
-    if (const auto& envVar = std::getenv("IE_VPU_MYRIADX")) {
-        return std::stoi(envVar) == 0;
-    }
-    return true;
-}
+extern bool CheckMyriad2();
```

Comment:
Why do you need `extern` here?

Reply:
Oh, I see, it is redundant (copy-past from deprecated test API). I'll remove it in the next PR with `DSR_*` tests enabling